site/en/docs/extensions/mv3/samples/index.md

```diff
@@ -1,9 +1,70 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Samples"
-#date: TODO
-#updated: TODO
-#description: TODO
+seoTitle: "Chrome Extension examples"
+date: 2020-12-11
+updated: 2022-12-16
+description: Intro to extension examples GitHub repository 
 ---
 
-Samples for Chrome Extensions are available [on GitHub](https://github.com/GoogleChrome/chrome-extensions-samples).
+## Overview {: #overview }
+
+The [Extensions sample GitHub
+repository][gh-samples] provides examples of extensions that address various use cases and call appropriate Chrome
+APIs. Use these to learn how extensions work or as a starting point for building your own extensions.
+
+## Browse by category {: #categories }
+
+The examples are currently located under the following directories:
+
+[api/][gh-api]
+: Extensions designed to demonstrate the capabilities of a specific API. For example, the [Action API example][gh-action] showcases extension IU elements such as the popup, tooltip, and badges among others. 
+
+[examples/][gh-examples]
+: Complete extensions that implement all the basic features for a given purpose.
+
+[tutorials/][gh-tutorials]
+: Examples covered in the [tutorials][gs-tutorials]. A few examples include [Tabs manager][tut-tabs-man], [Focus mode][tut-fm], and [Reading time][tut-rt].
```

Comment:
@jpmedley I adapted your suggestions to this list. Please verify that I understood correctly.

Reply:
Looks good!